<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>payTic</class>
 <widget class="QWidget" name="payTic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="AllpayLb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>共需支付：</string>
   </property>
  </widget>
  <widget class="QLabel" name="MypayLb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入支付金额：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="yuan1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
